src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/ShipmentState/orderShipmentState.html.twig

```diff
@@ -0,0 +1,15 @@
+{%
+    set viewOptions = {
+    cart: { icon: "adjust", color: "grey" } ,
+    cancelled: { icon: "ban", color: "red" } ,
+    shipped: { icon: "plane", color: "green" } ,
+    partially_shipped: { icon: "adjust", color: "yellow" } ,
+    ready: { icon: "clock", color: "blue" } ,
+}
```

Comment:
Wrong formatting (see [this file](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/PaymentState/orderPaymentState.html.twig))